src/features/bivariate_manager/components/BivariateMatrixControl/types.ts

```diff
@@ -1,22 +1,17 @@
 export type BivariateMatrixQuotientType = {
   id: [string, string];
   label?: string;
-  quality?: number;
+  quality?: string;
 };
 
 export type BivariateMatrixHeadingType = {
   label: string;
   selectedQuotient: BivariateMatrixQuotientType;
-  quality?: number;
+  quality?: string;
   quotients: BivariateMatrixQuotientType[];
 };
 
-export type CornerRange =
-  | 'good'
-  | 'bad'
-  | 'important'
-  | 'unimportant'
-  | 'neutral';
+export type CornerRange = 'good' | 'bad' | 'important' | 'unimportant' | 'neutral';
```

Comment:
this set is not supposed to be limited by front end?

Reply:
no, this type just describes possible values. 
on frontend we use values as just string, no checks is it a part of set or not.